pandas/io/common.py

```diff
@@ -345,6 +346,10 @@ def get_handle(
         Mode to open path_or_buf with.
     encoding : str or None
         Encoding to use.
+    errors : str, default 'strict'
+        Specifies how encoding and decoding errors are to be handled.
+        See the errors argument for :func:`open` for a full list
+        of options.
```

Comment:
this needs a versionadded tag

Reply:
added

Reply:
I need some help getting this tag right, the automated checks fail on this thing with a less than helpful message

Reply:
@roberthdevries you should put the versionadded directive below the errors argument you are adding. Also put a blank line before / after and indent one level further than where you currently have it